spec/request_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
```

Comment:
You did great job!
I can see your user request tests are working. However, we need a little more ajjustements for it :)

- [ ] For starting, please create a new folder that called `request` and put your request tests in it.